app/components/mandatenbeheer/bestuursorgaan-mandaten.hbs

```diff
@@ -9,14 +9,22 @@
     <t.content as |c|>
       <c.header>
         <th>Naam</th>
-        <th>Aantal Houders</th>
+        <th>Min aantal houders</th>
+        <th>Max aantal houders</th>
+        <th>Aantal houders</th>
         <th></th>
       </c.header>
       <c.body as |mandaat|>
         <td class="au-u-visible-small-up">
           {{mandaat.bestuursfunctie.label}}
         </td>
 
+        <td class="au-u-visible-small-up">
+          {{mandaat.minAantalHouders}}
```

Comment:
Should we hide it when its not set? This is only set in bestuursperiode 2025 - heden so all the other just show an empyt filed.  We could also set Ongekend text when not specified

![image](https://github.com/user-attachments/assets/23d547f2-7043-42e1-817b-27bd044de78e)

Reply:
good point, will do.